### PR TITLE
GP2-3071 Add docker setup/start script

### PR DESCRIPTION
**Pull request by froddd**
Makefile `secret` script now uses `sed` directly to create the new file to avoid using `-i` option, which ensures compatibility with both Mac and Linux platforms (and prevents creating extra `secrets-do-not-commit-e` file on Mac).

The secrets file is only created if it does not exist, otherwise user is prompted to delete if they require a fresh one.

CONTEXT: This should make onboarding and fresh setup slightly easier and more consistent when using Docker

### Workflow

- [x] Ticket exists in Jira https://uktrade.atlassian.net/browse/TICKET_ID_HERE
- [x] Jira ticket has the correct status.
- [ ] [Changelog](CHANGELOG.md) entry added.

### Reviewing help

- [x] ~~Explains how to test locally, including how to set up appropriate data~~
- [x] ~~Includes screenshot(s) - ideally before and after, but at least after~~

### Housekeeping

- [ ] ~~Added all new environment variables to Vault.~~
- [ ] ~~Cleaned up old feature flags~~
- [ ] ~~Upgraded any vulnerable dependencies.~~
- [ ] ~~Ran the `make manage download_geolocation_data` command~~
- [ ] ~~I have updated security dependencies~~
- [ ] ~~Python requirements have been re-compiled.~~
- [ ] ~~Frontend assets have been re-compiled.~~

### Merging

- [x] This PR can be merged by reviewers. (If unticked, please leave for the author to merge)

### CODE REVIEW

File: makefile [hunk 1]
```diff
@@ -95,8 +95,11 @@ install_requirements:
 	pre-commit install --install-hooks
 
 secrets:
```
Comment (depsiatwal) [inline makefile:97]: same are called secrets others are called init_secrets may want to make these consistent so it's easier to batch

**Review by depsiatwal — COMMENTED** [review]
see comment

File: makefile (hunk 1 above)
Comment (froddd) [reply makefile:97]: All makefiles in other linked repositories have been updated to `make secrets` for consistency -- and to make it easier to batch indeed!

Comment (depsiatwal) [reply makefile:97]: cool LGTM

**Review by depsiatwal — APPROVED** [review]
LGTM

**Review by depsiatwal — APPROVED** [review]
LGTM

**Pull request merged**